code/modules/reagents/oldchem/reagents/reagents_toxin.dm

```diff
@@ -26,6 +26,234 @@
 		return
 
 
+// Terror Spider, Green, Harmless Pheremone
+
+
+/datum/reagent/terror_green_toxin
```

Comment:
It's to keep it modular and more easily accessible. For the most part, 99% of the time, these reagents are only associated with the away mission and aren't generally used elsewhere--outside of the event).